examples/tfengine/resources_only.hcl

```diff
@@ -97,6 +97,7 @@ template "resources" {
       }]
       storage_buckets = [{
         name = "example-prod-bucket"
+        force_destroy = true
```

Comment:
can you add a comment above saying it's for example purpose only. Generally, it's not recommended as it would delete objects in the bucket if any...

Reply:
I think I can set it to false and add the # For example purposes only

It won't really affect since the default is false but it would be rendered during tests, which I think is the purpose of specifying it in at least one example.

Wdyt?

Reply:
sgtm

Reply:
Done